src/server/services/vergunningen-v2/helpers.test.ts

```diff
@@ -0,0 +1,398 @@
+import { CaseTypeV2 } from '../../../universal/types/vergunningen';
+import {
+  DecosZaakSource,
+  TouringcarDagontheffing,
+  WerkzaamhedenEnVervoerOpStraat,
+} from './config-and-types';
+import { decosZaakTransformers } from './decos-zaken';
+import {
+  getCustomTitleForVergunningWithLicensePlates,
+  getDecosZaakTypeFromSource,
+  hasInvalidDecision,
+  hasOtherActualVergunningOfSameType,
+  isExcludedFromTransformation,
+  isExpired,
+  isNearEndDate,
+  isScheduledForRemoval,
+  isWaitingForPaymentConfirmation,
+  toDateFormatted,
+  transformBoolean,
+  transformKenteken,
+} from './helpers';
+
+describe('helpers/Vergunningen', () => {
+  vi.useFakeTimers();
+  vi.setSystemTime(new Date('2022-10-06'));
+
+  test('isNearEndDate', () => {
+    {
+      // No end date
+      const vergunning: any = {
```

Comment:
Is `null` niet een reden om een error op te gooien? Anders zeg je dat null niet dichtbij de eind datum ligt terwijl het geen eind datum is. Lijkt me misschien beter dat de caller altijd een goede datum hierin moet stoppen en anders dit moet afhandelen. Ik zie in andere bestanden dit meestal in een `&&` expressie. Misschien is het beter daar dan de input bij te zetten zo dat het explicieter word dat als deze waarde niet bestaat dan dat die expressie `false` word zonder dat je die internals van die functie hoeft te lezen.

Reply:
Ik denk 't niet. Het is een helper gemaakt voor vergunningen, en het hoofdargument is ook de vergunning. 

Als dit een hele generieke functie zou zijn buiten de context van vergunningen zou je misschien wat expliciter kunnen zijn